network/mounts.csv

```diff
@@ -1,9 +1,9 @@
 Mount,Network,Name,Latitude,Longitude,Elevation,Datum,Description,Start Date,End Date
 CHTB,BC,Christchurch Telecom Building,-43.53162,172.63544,0,WGS84,Images of Christchurch.,2011-11-28T00:00:03Z,2019-02-24T21:00:00Z
-DISC,VC,Discovery Lodge,-39.156143224,175.491281443,9999,WGS84,Images of Ruapehu and Ngauruhoe.,2020-07-28T02:20:00Z,9999-01-01T00:00:00Z
+DISC,VC,Discovery Lodge,-39.16732,175.48055,9999,WGS84,Images of Ruapehu and Ngauruhoe.,2020-07-28T02:20:00Z,9999-01-01T00:00:00Z
 KAKA,VC,Kakaramea,-38.97490003,175.694834619,1327,WGS84,Images of Tongariro.,2011-09-05T00:01:00Z,9999-01-01T00:00:00Z
 KMTP,VC,Kaimanawa Tai Ping,-39.29642,175.7663,1366,WGS84,Images of Ruapehu and Ngauruhoe.,2009-03-03T02:00:00Z,9999-01-01T00:00:00Z
-MNTA,VC,Mountain Air,-39.156143224,175.491281443,9999,WGS84,Images of Ruapehu and Ngauruhoe.,2000-11-14T13:00:10Z,9999-01-01T00:00:00Z
+MNTA,VC,Mountain Air,-39.156143224,175.491281443,9999,WGS84,Images of Ruapehu and Ngauruhoe.,2000-11-14T13:00:10Z,2022-07-28T00:00:00Z
```

Comment:
for MNTA, the last installed equipment entry in `install/cameras.csv` is `2020-07-27T21:20:00Z`
is 2022 a typo here?